tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc

```diff
@@ -40,36 +40,36 @@ private ComboBox<CustomerStatus> status = new ComboBox<>("Status");
 private DatePicker birthDate = new DatePicker("Birthdate");
 ----
 
-The form also needs two buttons for _saving_ and _deleting_ `Customer` instances. Add them to the `CustomerForm` class as follows:
-
+. The form also needs two buttons to save and delete `Customer` instances. Add the buttons to the `CustomerForm` class as follows:
++
 [source,java]
 ----
 private Button save = new Button("Save");
 private Button delete = new Button("Delete");
 ----
 
-With the components in place, you can set further configurations and add them to the form in a constructor. Add the following constructor to the `CustomerForm` class:
-
+. With the components in place, you can set further configurations and add them to the form in a constructor. Add the following constructor to the `CustomerForm` class:
++
 [source,java]
 ----
 public CustomerForm() {
-    status.setItems(CustomerStatus.values()); <1>
+    status.setItems(CustomerStatus.values()); 
 
     HorizontalLayout buttons = new HorizontalLayout(save, delete);
-    save.addThemeVariants(ButtonVariant.LUMO_PRIMARY); <2>
+    save.addThemeVariants(ButtonVariant.LUMO_PRIMARY); 
     add(firstName, lastName, status, birthDate, buttons);
 }
 ----
-<1> Adds all the enum values as options to the `ComboBox`.
+* `status.setItems` adds all the enum values as options to the `ComboBox`.
 
-<2> Makes the save button prominent by decorating it with a style name.
+* `addThemeVariants` makes the save button prominent by decorating it with a style name.
 
-== Showing and Hiding the Form
+== Showing and Hiding the Form Data
```

Comment:
@ruthstocks "Form Data" sounds wrong. What the tutorial is talking about is a "Form". Can you please fix before a merge this?

Reply:
Sorry about the delay. Fixing now.